docs/manuals/verification/mirror_rb.rst

```diff
@@ -0,0 +1,328 @@
+distance :math:`k` away from the expected output bit string (note :math:`h_0 = P`). We
+also compute the **effective polarization**, which is fitted and visualized by default:
+
+.. math::
+
+    S = \frac{4^n P_0}{4^n - 1} - \frac{1}{4^n - 1}.
+
+In [1]_, the function :math:`A\alpha^\ell` (without a baseline) is fit to the 
+effective polarizations to find entanglement infidelities.
+
+In Qiskit Experiments, mirror RB analysis results include the following:
+
+- ``alpha``: the depolarizing parameter. The user can select which of :math:`P, P_0, S` 
+  to fit, and the corresponding :math:`\alpha` will be provided.
+
+- ``EPC``: the expectation of the average gate infidelity of a layer sampled 
+  according to :math:`\Omega`.
+
+- ``EI``: the expectation of the entanglement infidelity of a layer sampled 
+  according to :math:`\Omega`. 
+
+Note that the ``EPC`` :math:`\epsilon_a` and the ``EI`` :math:`\epsilon_e` are 
+related by 
+
+.. math:: 
+
+    \epsilon_e = \left(1 + \frac{1}{2^n}\right) \epsilon_a, 
+
+where :math:`n` is the number of qubits (see [2]_). 
+
+
+Running a mirror RB experiment
+------------------------------
+
+The distribution for sampling layers, :math:`\Omega`, must be specified by the user when
+instantiating a mirror RB experiment. A commonly used :math:`\Omega` is one generated by
+the **edge grab** algorithm [3]_. The Clifford layers in :math:`\mathbb{L}` are
+constructed from a gate set consisting of one-qubit Clifford gates and a single
+two-qubit Clifford gate (e.g., CX) that can be applied to any two connected qubits. The
+user can specify an expected two-qubit gate density :math:`\xi \in \left[0,
+\frac{1}{2}\right]`, and each intermediate Clifford layer will have approximately
+:math:`n \xi` CXs on average.
+
+Even though a :class:`.MirrorRB` experiment can be instantiated without a backend, the
+backend must be specified when the circuits are sampled because :math:`\Omega` depends
+on the backend's connectivity. To use your own :math:`\Omega`, you have to implement
+your own subclass of the abstract :class:`.BaseSampler` class, but here we will use
+the built-in :class:`.EdgeGrabSampler`. Here's how to instantiate and run the
+experiment:
+
+.. jupyter-execute::
+
+    import numpy as np
+    from qiskit_experiments.library import MirrorRB
+    from qiskit_experiments.library.randomized_benchmarking.sampling_utils import EdgeGrabSampler
+
+    from qiskit_aer import AerSimulator
+    from qiskit.providers.fake_provider import FakeParis
+    
+    backend = AerSimulator.from_backend(FakeParis())
+
+    lengths = np.arange(2, 810, 200)
+    num_samples = 5
+    seed = 1010
+    qubits = (0,1)
+
+    exp_2q = MirrorRB(qubits, lengths, backend=backend, num_samples=num_samples, seed=seed)
+    expdata_2q = exp_2q.run(backend).block_for_results()
+    results_2q = expdata_2q.analysis_results()
+
+.. jupyter-execute::
+
+    print("Gate error ratio: %s" % expdata_2q.experiment.analysis.options.gate_error_ratio)
```

Comment:
Good point, actually this line was copied from the RB manual so we should update that line. I'll remove this line since it doesn't show any new information.